user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Item.md

```diff
@@ -0,0 +1,114 @@
+
+> [!IMPORTANT]
+> You need to have the "Owner" role in order to access/create organization keys. See [Changing the role of a dataminer.services user](xref:Changing_the_role_of_a_DCP_user) for information on how to change a role for a user.
+
+1. In the Admin app, under *Organization* in the sidebar on the left, select the *Keys* page.
+
+1. At the top of the page, click *New Key*.
+
+1. Configure the key with a label of your choice and the permission *Register catalog items*.
+
+   ![Organization Key](~/user-guide/images/tutorial_catalog_registration_create_org_key.png)
+
+1. Copy the key and use it as the value in the **Ocp-Apim-Subscription-Key** header.
+
+   ![Register item HTTP header](~/user-guide/images/tutorial_catalog_registration_urlandheaders.png)
+
+## Step 3: Register the Catalog item body
+
+1. Create a .zip file containing the following items:
+
+   ![zip content structure](~/user-guide/images/tutorial_catalog_registration_item_zip_structure.png)
+
+   - A *README.md* file. This *README.md* will be shown in the description tab in the Catalog.
+
+     Copy the snippet below, changing the content as you see fit using your favorite editor, and save the file as `README.md`.
+
+     ```md
+     # Example_Rates_Custom
+
+     This is a connector that serves as an example for how to integrate bit rates.
+
+     ## About
+
+     This Catalog item has been made available as part of the [Catalog registration tutorial](https://docs.dataminer.services/tutorials/Tutorials.html)
+     ```
+
+   - Optionally, if you want to use images in the readme, an images folder containing those images.
+
+   - A *manifest.yml* file describing the properties of the Catalog item.
+
+     Copy the snippet below, changing the fields as you see fit using your favorite editor, and save the file as `manifest.yaml`.
+
+     > [!NOTE]
+     > Make sure to obtain a new GUID to uniquely store and identify the Catalog item. To do so, you can for instance navigate to [GUID generator](https://guidgenerator.com/) and create a new GUID using the format *hyphens*, or you can use your own preferred method. 
+
+     ```yml
+     type: 'connector'
+
+     id: '1742495c-9231-4eeb-a56e-1fec8189246e'
+
+     title: 'My Catalog Registration Example'
+
+     short_description: 'Example connector showing how to calculate bitrates and other rates on any changing numeric data.'
+
+     source_code_url: 'https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom'
+
+     owners:
+       - name: 'john doe'
+         email: 'john.doe@skyline.be'
+     tags:
+       - 'example'
+     ```
+
+1. Add the zip file as the value of the body of the request as shown below.
+
+   The body of the request needs to be in the **multipart/form-data** format with a key of type **File** with the name **file**.
+
+   ![Register item http body](~/user-guide/images/tutorial_catalog_registration_item_body.png)
```

Comment:
Could you also make a new version of this screenshot from a smaller window (ideally about the width of the main text area on Docs), so it's easier for users to see the text?

Reply:
updated images, also for version tutorial